doc/nrf/device_guides/nrf54l.rst

```diff
@@ -0,0 +1,24 @@
+.. _ug_nrf54l:
+
+Working with nRF54L Series
+##########################
+
+Zephyr and the |NCS| provide support and contain board definitions for developing on the following nRF54L Series devices:
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK
+     - PCA number
+     - Build target
+     - Documentation
+   * - :ref:`zephyr:nrf54l15pdk_nrf54l15`
+     - PCA10156
+     - ``nrf54l15pdk_nrf54l15@soc1``
```

Comment:
Target will be **nrf54l15pdk_nrf54l15_cpuapp** or **nrf54l15pdk_nrf54l15_cpuapp@0.3.0** accordingly to revision of PDK available if I remember correctly.

Reply:
Or at least without **soc1** because I see that **_cpuapp** part is not included in other devices as well